tests/test_wmb.py

```diff
@@ -0,0 +1,151 @@
+import sys
+import pytest
+import pytz
+import csv
+import random
+from sqlalchemy import and_
+
+from datetime import datetime
+from decimal import Decimal
+from io import StringIO
+from tempfile import TemporaryFile
+
+from pycds import Obs, History, Network, Station
+from crmprtd.wmb import ObsProcessor, insert_obs, check_history
+
+
+@pytest.mark.parametrize(('val', 'hid', 'd', 'vars_id', 'expected'), [
+    # It looks like the vars table is indexed?
+    (2.7, 20, datetime(2016, 9, 13, 6, tzinfo=pytz.utc), Decimal(1), 4)
+])
+def test_insert_obs(test_session, val, hid, d, vars_id, expected):
+    insert_obs(val, hid, d, vars_id, test_session)
+    q = test_session.query(Obs)
+    assert q.count() == expected
+    # sanity check
+    q = test_session.query(Obs).filter(Obs.datum == val, Obs.history_id == hid, Obs.time == d, Obs.vars_id == vars_id)
```

Comment:
This is a fine sanity check. The other way to do it would be to filter on history_id, time, and vars_id and then assert that datum == val (since datum is essentially the one item that is recorded... i.e. the "independent variable").